### PR TITLE
Added support for course modules

### LINKED ISSUE #106: Support for purchasing single modules for a course.
We need the ability to purchase 1 or more modules inside a course in edx. Dependent upon https://github.com/mitocw/edx-platform/pull/173

### PR DESCRIPTION
**Pull request by giocalitri**
Fixes #106

### CODE REVIEW

File: courses/views.py [hunk 1]
```diff
@@ -128,6 +128,7 @@ def create_ccx(request):
         user_email (string): email address of the user to make CCX Coach
         display_name (string): Title of the CCX
         max_students_allowed (int): Maximum number of students to allow into the ccx
```
Comment (noisecapella) [inline courses/views.py]: When you're refactoring can you change this to `total_seats`?

Comment (giocalitri) [inline courses/views.py]: refactoring what?

Comment (noisecapella) [inline courses/views.py]: `max_students_allowed` should be `total_seats`

Comment (giocalitri) [inline courses/views.py]: `max_students_allowed` is the name in edx

Comment (justinabrahms) [inline courses/views.py]: yes, but on line 156, we expect it to be `total_seats` when coming from the `POST`.

Comment (giocalitri) [inline courses/views.py]: sorry, I did not understand the point until now

**Comment by noisecapella** [comment]
This looks good overall but it needs to be changed to use `request.json()` instead of `request.POST`.

**Comment by giocalitri** [comment]
didn't discuss this yesterday morning (Monday) and said that we merge it as it is and then we convert it?

**Comment by noisecapella** [comment]
Fine with me :+1:

**Comment by giocalitri** [comment]
merging

**Pull request merged**